ds/data/processed/aps/aps_nvq4_education_data.nuts3.yaml

```diff
@@ -0,0 +1,31 @@
+api_doc_url: https://www.nomisweb.co.uk/api/v01/help
+api_type: FETCH
+description_short: Economically active in professionals
+description: Percentage of economically active persons in professional occupations by NUTS 3 regions.
+endpoint_url: {
+  2010: https://www.nomisweb.co.uk/api/v01/dataset/NM_17_5.jsonstat.json?geography=1912602625...1912602758&date=latestMINUS27,latestMINUS23,latestMINUS19&variable=353&measures=20599,21001,21002,21003,
+  2013: https://www.nomisweb.co.uk/api/v01/dataset/NM_17_5.jsonstat.json?geography=1883242497...1883242664&date=latestMINUS15,latestMINUS11,latestMINUS7&variable=353&measures=20599,21001,21002,21003,
+  2016: https://www.nomisweb.co.uk/api/v01/dataset/NM_17_5.jsonstat.json?geography=1832910849...1832911016&date=latestMINUS3&variable=353&measures=20599,21001,21002,21003
+  } # nuts_spec_year: endpoint_url
+framework_group: public_rnd_capability
+is_experimental: False
+order: [year, nuts_id, nuts_year_spec, value.id] # do not change; specifies the order of the exported indicator fields
+region:
+  type: NutsRegion # the region type, e.g. NutsRegion or LepRegion
+  level: 3
+  source: PROVIDED
+schema:
+  nuts_id:
+    type: NutsRegion.nuts_id
+  nuts_year_spec:
+    type: NutsRegion.nuts_year_spec
+  value:
+    description: Percentage of population employed in professional occupations.
+    format: .1

```

Comment:
This should be `format: .1f` (we got it wrong originally, I'm changing these in an upcoming PR)